### DISCUSSION TITLE
Risk-Adjusted Return Reward Function

### DISCUSSION THREAD

**Discussion by nickjtay** [AI4Finance-Foundation/FinRL#848] [Q&A]
Quick question--I am wondering why the tutorials I've seen and the repo use cumulative returns as the reward function instead of a risk adjusted measure such as the sharpe ratio? Are there any examples of this being done within the FinRL framework? I'm interested in experimenting with changes to the reward function, and other hyperparameters, and would love to understand other peoples thoughts and experiences around this topic.

**Comment by zhumingpassional** [comment]
we use the change of total_asset as reward, not cumulative returns.

you can add sharpe ratio to reward function. you can try
